README.md

```diff
@@ -131,6 +136,7 @@ Start tests with verbose flag:
 ```
 
 ## Key bindings
+<a name="keys"></a>
```

Comment:
there is no need to add this element here, you can use `#key-bindings` above to link to this section